### DISCUSSION TITLE
Feature Request: Option for Flyout Modal to Appear from Left Side

### DISCUSSION THREAD

**Discussion by ricardo-lobo** [livewire/flux#600] [Feature Requests] [answered]
Hello,

Could you please add an option to the Flyout Modal component (https://fluxui.dev/components/modal#flyout) to allow it to appear from the left side of the screen as well as the right?

Thank you!

**Comment by calebporzio** [comment, marked as answer]
This will be in the next release. Thanks!

![CleanShot 2024-11-30 at 22 18 57@2x](https://github.com/user-attachments/assets/792d77f5-a398-4da3-9f78-884c8d3d8103)
